server/interfaces.coffee

```diff
@@ -32,4 +35,25 @@ Meteor.methods
         offset = UTCOffsets[tz]
         dateUTC = match[1].replace(' ', 'T') + offset
         dateUTC
-    )
+
+  queryForSuggestedArticles: (eventId) ->
```

Comment:
Could you make this remove suggestions that have already been added to the event?